src/main/java/edu/iastate/ece/sd/sdmay2126/application/FBAParameters.java

```diff
@@ -32,7 +34,7 @@ public FBAParameters(boolean fluxVariabilityAnalysis, boolean minimizeFlux, bool
                          float activationCoefficient, float maxCarbonUptake, float maxNitrogenUptake,
                          float maxPhosphateUptake, float maxSulfurUptake, float maxOxygenUptake,
                          String reactionToMaximize, float expressionThreshold, float expressionUncertainty,
-                         String geneKnockouts) {
+                         String geneKnockouts, String customFluxBounds) {
```

Comment:
This value is never used to initialize the custom flux bounds when reading in from a file.

Custom flux bounds needs to be initialized or it could cause null pointers later in the program.